Tests/CommandsTests/TestToolTests.swift

```diff
@@ -181,6 +181,8 @@ final class TestToolTests: CommandsTestCase {
 
     func testEnableTestDiscoveryDeprecation() throws {
         let compilerDiagnosticFlags = ["-Xswiftc", "-Xfrontend", "-Xswiftc", "-Rmodule-interface-rebuild"]
+        let defaultTestEntryPointName = try XCTUnwrap(SwiftTarget.testEntryPointNames.first)
```

Comment:
consider a private/internal extension on SwiftTarget to help with keeping this DRY for the scope of the tests.

Reply:
I found a simpler overall approach: decompose the list of entry point file names into two properties, where one returns the default one (`XCTMain.swift`), so we don't need any conditional unwrapping boilerplate

```swift
    /// The default name for the test entry point file located in a package.
    public static let defaultTestEntryPointName = "XCTMain.swift"

    /// The list of all supported names for the test entry point file located in a package.
    public static var testEntryPointNames: [String] {
        [defaultTestEntryPointName, "LinuxMain.swift"]
    }
```